leakcanary-sample/src/main/java/com/example/leakcanary/MainTvActivity.java

```diff
@@ -0,0 +1,39 @@
+package com.example.leakcanary;
+
+import android.app.Activity;
+import android.content.Intent;
+import android.os.AsyncTask;
+import android.os.Bundle;
+import android.os.SystemClock;
+import android.view.View;
+
+public class MainTvActivity extends Activity {
```

Comment:
Why do we need a new MainTvActivity, why not add the intent filter to the existing activity?